internal/cpp/rag_analyzer.cpp

```
@@ -696,14 +696,14 @@ int32_t RAGAnalyzer::Load() {
     if (!fs::exists(pos_def_path)) {
         printf("Invalid post file: %s", pos_def_path.string().c_str());
         // return Status::InvalidAnalyzerFile(pos_def_path);
-        return -1;
+        return -2;
     }
     own_dict_ = true;
     trie_ = new DartsTrie();
     pos_table_ = new POSTable(pos_def_path.string());
```

Cppcheck (2.21.0)

[error] Unhandled exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)